docs/faq.rst

```diff
@@ -12,6 +12,39 @@ Yes, the `deduplication`_ technique used by
 |project_name| makes sure only the modified parts of the file are stored.
 Also, we have optional simple sparse file support for extract.
 
+If you use tools like Borg (or rsync, tar, cp and most other backup tools) to back up
+virtual machines, then these should be turned off for doing so, since Borg doesn't do
+*any* kind of snapshotting on its own. Backing up live VMs this way can (and will)
+result in corrupted or inconsistent backup contents: a VM image is just a regular file to
+Borg with the same issues as regular files when it comes to concurrent reading and writing from
+the same file.
+
+A better method is to use file system snapshots on the VM host, which establishes
+crash-consistency for the VM images. This means that with most file systems
+(that are journaling) the FS will always be fine in the backup (but may need a
+journal replay to become accessible).
+
+Usually this does not mean that file *contents* on the VM are consistent, since file
+contents are normally not journaled. Notable exceptions are ext4 in data=journal mode or ZFS.
```

Comment:
and that they have a special nodatacow file attribute to be used with databases that do their own journaling. I don‘t know is zfs has a similar attribute.
I think we should mention btrfs explicitly because it is getting popular.